simple_parameter_store.gemspec

```diff
@@ -1,42 +1,34 @@
+# frozen_string_literal: true
 
-lib = File.expand_path("../lib", __FILE__)
+lib = File.expand_path('lib', __dir__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require "simple_parameter_store/version"
+require 'simple_parameter_store/version'
 
 Gem::Specification.new do |spec|
-  spec.name          = "simple_parameter_store"
+  spec.name          = 'simple_parameter_store'
   spec.version       = SimpleParameterStore::VERSION
-  spec.authors       = ["Timo Schilling"]
-  spec.email         = ["timo@schilling.io"]
+  spec.authors       = ['Timo Schilling']
+  spec.email         = ['timo@schilling.io']
```

Comment:
Since you moved the gem over, does it make sense to use your corporate address here (just a suggestion, feel free not to).